src/chronometer.js

```diff
@@ -1,33 +1,62 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+    }
+   currentTime = 0
+    intervalId = null
+
+
+  start(printTimeCallback){
+    let currentTime = 0;
 
-  start(printTimeCallback) {
-    // ... your code goes here
+function start(printTimeCallback) {
+  let currentTime = 0;
+
+function start(printTimeCallback) {
+    currentTime = 0;
+    let iidInterval = setInterval(() => {
+        currentTime++;
```

Comment:
Falta agregar el this a la propiedad (`this.currentTime++`) para indicar que es propiedad de la clase.